src/generators/new/infra/config/index.js

```diff
@@ -27,6 +27,14 @@ module.exports = async ({ template: { generate }, parameters: { options } }) =>
     requires.database = 'require(\'./postgres\')'
   }
 
+  if (options.sqlserver) {
```

Comment:
the database options is growing fat, I think we can refactpry this code like:
``` js
for(const db of ['postgres', 'mongo'm 'sqlserver']){
   if(!options(db]) continue
   await generate({
      template: `infra/config/${db}\.ejs`,
      target: `src/infra/config/${db}\.js`
    })
    requires.database = `require(\'./${db}\')`

Reply:
done